packages/cli-kit/src/environment/local.ts

```diff
@@ -154,7 +160,12 @@ export function ciPlatform(env = process.env): {isCI: true; name: string} | {isC
  * Gets info on the Web IDE platform the CLI is running on, if applicable
  */
 export function webIDEPlatform(env = process.env) {
```

Comment:
`webIDEPlatform` and `cloudEnvironment` seem somewhat related. The former is a subset of the data returned by the latter. What about having a single function and adjusting the return type to allow filtering:

```js
{ platform: 'codespaces', editor: false },
{ platform: 'gitpod', editor: true }
{ platform: 'spin', editor: false }
```